Apps/phdnsdb/dnsdb.json

```diff
@@ -1,413 +1,1072 @@
-          "order": 3,
-          "data_type": "string"
-        },
-        "limit": {
-          "required": false,
-          "description": "Max records to return",
-          "order": 4,
-          "data_type": "numeric",
-          "default": 200
-        }
-      },
-      "versions": "EQ(*)",
-      "render": {
-        "width": 12,
-        "title": "IP Lookup",
-        "type": "custom",
-        "view": "dnsdb_view.display_lookup_info",
-        "height": 5
-      },
-      "output": [
-        {
-          "data_path": "action_result.data.*.count",
-          "data_type": "numeric"
-        },
-        {
-          "data_path": "action_result.data.*.rdata",
-          "data_type": "string",
-          "contains": [
-            "ip",
-            "ipv6"
-          ]
-        },
-        {
-          "data_path": "action_result.data.*.rrname",
-          "data_type": "string",
-          "contains": [
-            "domain"
-          ]
-        },
-        {
-          "data_path": "action_result.data.*.rrtype",
-          "data_type": "string"
-        },
-        {
-          "data_path": "action_result.data.*.time_last",
-          "data_type": "numeric"
-        },
-        {
-          "data_path": "action_result.data.*.time_first",
-          "data_type": "numeric"
-        },
-        {
-          "data_path": "action_result.data.*.zone_time_last",
-          "data_type": "numeric"
-        },
-        {
-          "data_path": "action_result.data.*.zone_time_first",
-          "data_type": "numeric"
-        },
-        {
-          "data_path": "action_result.status",
-          "data_type": "string"
-        },
-        {
-          "data_path": "action_result.message",
-          "data_type": "string"
-        },
-        {
-          "data_path": "action_result.parameter.ip",
-          "data_type": "string",
-          "contains": [
-            "ip",
-            "ipv6"
-          ]
-        },
-        {
-          "data_path": "action_result.parameter.network_prefix",
-          "data_type": "numeric"
-        },
-        {
-          "data_path": "action_result.parameter.record_seen_after",
-          "data_type": "string"
-        },
-        {
-          "data_path": "action_result.parameter.record_seen_before",
-          "data_type": "string"
-        },
-        {
-          "data_path": "action_result.parameter.limit",
-          "data_type": "numeric"
-        },
-        {
-          "data_path": "action_result.summary.total_domains",
-          "data_type": "numeric"
-        },
-        {
-          "data_path": "summary.total_objects",
-          "data_type": "numeric"
-        },
-        {
-          "data_path": "summary.total_objects_successful",
-          "data_type": "numeric"
+    "appid": "adf140f0-60e3-45ff-8d69-258b3c0a63b1",
+    "name": "DNSDB",
+    "description": "This app supports investigative DNS lookup actions on DNSDB",
+    "publisher": "Farisght Security",
```

Comment:
@cschmidt0121 Hey, could you please check the publisher value?

Reply:
My mistake. Thanks for fixing that; it looks good now.